### DISCUSSION TITLE
Tricky Store module compatibly

### DISCUSSION THREAD

**Discussion by IslamIssa1997** [salvogiangri/UN1CA#155] [Feature requests]
hello mr.salvo

first of all thank you million times for your effort in this project and all of your contributions to make smart phones even smarter (LOVE YOU BROTHER ❤️)

My request is (if it code able) to make this awsome OS to be able to run along or implement (sorry for such bad english) with Tricky Store module to pass strong (already did it with stock rom a52s 5G with pif alongside with lecked keybox.xml ) and if it possible will this make samsung pass usable again with knox patch ? link for tricky store ( https://t.me/real5ec1cff/229 )

and one more thing if i rooted and make all hide methods probably native test app show partition check fail app link ( https://t.me/LSPosed/279 ) (( sorry if this question not related to your OS )) but is this something i did wrong or some fail from device or that a real detection for magisk modifications (cause it pass strong and all other apps working good except for one bank app wich i tried on another device using stock never unlocked bootloader

sorry for taking that much of words and hope you get a proper time to respond

hope you good health and good life ❤️

**Comment by salvogiangri** [comment]
A toggle to disable the integrated PIF in ROM has been added and will be available from the next release.

<img loading="lazy" src="https://github.com/user-attachments/assets/963e023e-1c6c-4201-ae6c-e6e675568210" width="30%"/><br>

Samsung Pass and Wallet/Pay do not use key attestation to check for the device integrity, instead they check for those values by fetching them directly from ICCC/bootloader in their TAs, so there's no way to get those app to work on warranty void devices (https://github.com/salvogiangri/KnoxPatch/issues/43).

**Reply by IslamIssa1997** [reply]
thanks for replying so fast brother 
would you please also add a toggle for [FrameworkPatch] chiteroman cause i think it conflicts with tricky store module or something
thanks

**Reply by salvogiangri** [reply]
The toggle already enables/disables both PIF and FrameworkPatch to allow using custom modules to handle key attestation and GMS spoofing.

**Reply by IslamIssa1997** [reply]
thanks mr.salvo i hope you keep up with a good spirit 

i thought you have quite developing  😭  i accidently hit with your git repos and i was delighted when sawing a commits by your name

don't retire again bro ❤️

**Reply by IslamIssa1997** [reply]
sorry for this very noob question

could anyone use keybuster to abtain keybox from a device or exploit trustzone like you said here ( https://github.com/salvogiangri/KnoxPatch/issues/43#issuecomment-1704612908 ) cause in keybuster readme i think iam still lost there

**Reply by IslamIssa1997** [reply]
![image](https://github.com/user-attachments/assets/46d5a849-f533-49b4-80fc-7870c090dd48)

i think you maybe deleted released part but still in readme

**Reply by salvogiangri** [reply]
> could anyone use keybuster to abtain keybox from a device or exploit trustzone like you said here ( https://github.com/salvogiangri/KnoxPatch/issues/43#issuecomment-1704612908 ) cause in keybuster readme i think iam still lost there 

That's what I actually tried to do! Unfortunately tho this isn't enough. A TEE exploit would be required in order to obtain the GAK private key.

**Reply by IslamIssa1997** [reply]
is a TEE exploit tricky to achieve or impossible!!!

> That's what I actually tried to do!

i hope you achieve this soon
